defold-spine/editor/src/spineext.clj

```diff
@@ -668,10 +668,18 @@
    :atlas (resource/resource->proj-path atlas-resource)})
 
 
-(g/defnk produce-spine-scene-own-build-errors [_node-id atlas spine-json]
+(g/defnk produce-spine-scene-own-build-errors [_node-id atlas spine-json texture-set-pb spine-json-content]
   (g/package-errors _node-id
                     (validate-scene-atlas _node-id atlas)
-                    (validate-scene-spine-json _node-id spine-json)))
+                    (validate-scene-spine-json _node-id spine-json)
+                    (when (and texture-set-pb spine-json-content)
+                      (try
+                        (plugin-load-file-from-buffer
```

Comment:
The pointer calls `SPINE_Destroy` in its `finalize` method, so we can rely on GC here.